src/com/seafile/seadroid2/ui/activity/UnlockGesturePasswordActivity.java

```diff
@@ -71,6 +69,12 @@ protected void onDestroy() {
             mCountdownTimer.cancel();
     }
 
+    @Override
+    public void onBackPressed() {
+        // stop default action (finishing the current activity) to be executed.
+        // super.onBackPressed();
+    }
+
```

Comment:
This can fix the gesture lock problem, but It's really not a good idea to do nothing when the user presses the back button... When the user presses the back button, he means to exit the app and go back to the home screen.